src/graphics/basic_components/table.ts

```diff
@@ -15,6 +15,7 @@ export interface TableOptions {
 export class Table {
   private table: HTMLTableElement;
   private tbody: HTMLTableSectionElement;
+  tableWrapper: HTMLDivElement;
```

Comment:
Can we make this private?

```suggestion
  private tableWrapper: HTMLDivElement;
```